tools/interop_matrix/run_interop_matrix_tests.py

```diff
@@ -68,6 +68,13 @@
                   default='report.xml',
                   help='The result file to create.')
 
+argp.add_argument('--allow_flakes',
+                  default=True,
```

Comment:
I think the default should be `False`, to be consistent with our other test execution scripts

Reply:
It doesn't really need to be the same as other tests, since the flakiness is pretty high for this particular suite. However, I'm ok either way, so updated.